drivers/sensor/lsm6dsl/lsm6dsl_spi.c

```diff
@@ -71,12 +82,17 @@ static int lsm6dsl_raw_write(struct spi_config *spi_cfg, u8_t reg_addr,
 			.len = len,
 		}
 	};
+	const struct spi_buf_set tx = {
+		.buffers = tx_buf,
+		.count = 1
```

Comment:
shouldn't be this one equals to '2'?

Reply:
yep, I should buy new glasses (can't change the brain so...)

Reply:
@tbursztyka 
OK, thx!
The acc/gyro stuff was working fine.
Nevertheless I'm now re-testing (prior to submit) the new sensorhub part, and your changes
are not working anymore. If I put '2' there everything works.

Reply:
pushed a fixed version